.travis.yml

```diff
@@ -56,7 +57,7 @@ script:
   - test -z $(gofmt -s -l $GO_FILES)         # Fail if a .go file hasn't been formatted with gofmt
   - go test --timeout 5s -v -race ./...      # Run all the tests with the race detector enabled
   - ./testAndCover.sh                        # Run the tests again to get coverage info
-  - go vet ./...                             # go vet is the official Go static analyzer
+  - go vet -composites=false ./...           # go vet is the official Go static analyzer
```

Comment:
why disabling composite keys here?

Reply:
because the tests at linkedin were failing https://github.com/linkedin/Burrow/pull/450
I can't remember exactly the error message